[DOCS] Add collapsible sections to 8.0 breaking changes [Part 1] (#55811)
elastic/elasticsearch: docs/reference/migration/migrate_8_0.asciidoc

//Installation and Upgrade Guide

//tag::notable-breaking-changes[]

.Indices created in {es} 7.0 and earlier versions are not supported.
[%collapsible]
====
*Details* +
Elasticsearch 8.0 can read indices created in version 7.0 or above.  An
Elasticsearch 8.0 node will not start in the presence of indices created in a
version of Elasticsearch before 7.0.

*Impact* +

> Indices created in Elasticsearch 6.x or before will need to be reindexed with
> Elasticsearch 7.x in order to be readable by Elasticsearch 8.x.

Reviewer: ```suggestion
Reindex indices created in {es} 6.x or before with {es} 7.x if they need to be carried forward to  {es} 8.x. 
```
